packages/suite/src/views/wallet/trading/common/TradingForm/TradingFormOfferOTC.tsx

```diff
@@ -76,19 +117,32 @@ export const TradingFormOfferOTC = () => {
                             : 'TR_TRADING_OTC_INFO_SELL'
                     }
                     values={{
-                        minimumFiat: localizeNumber(otcData.minimumFiat, locale),
-                        fiatSymbol: currencySelect.toUpperCase(),
+                        minimumFiat: localizeNumber(displayedFiatLimit, locale),
+                        fiatSymbol: displayedFiatCurrency.toUpperCase(),
                     }}
                 />{' '}
-                <TrezorLink href={apiUrl.toString()} target="_blank" typographyStyle="hint">
-                    <Translation
-                        id={
-                            context.type === 'buy'
-                                ? 'TR_TRADING_OTC_LINK_BUY'
-                                : 'TR_TRADING_OTC_LINK_SELL'
-                        }
-                    />
-                </TrezorLink>
+                {links.map((link, index) => (
+                    <Fragment key={index}>
+                        <TrezorLink href={link.url} target="_blank" typographyStyle="hint">
+                            <Translation
+                                id={
+                                    context.type === 'buy'
+                                        ? 'TR_TRADING_OTC_LINK_BUY'
+                                        : 'TR_TRADING_OTC_LINK_SELL'
+                                }
+                                values={{
+                                    providerName: link.name,
+                                }}
+                            />
+                        </TrezorLink>
+                        {index < links.length - 1 && (
+                            <>
+                                {' '}
+                                <Translation id="TR_TRADING_OTC_LINK_OR" />{' '}
+                            </>
+                        )}
```

Comment:
Can you please check if you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat ? I don't like this "or" string